src/backend/libs/efiling-account-client/src/main/java/ca/bc/gov/open/jag/efilingaccountclient/config/AutoConfiguration.java

```diff
@@ -39,15 +40,22 @@ public RoleRegistryPortType roleRegistryPortType() {
        return getPort(Clients.ROLE, RoleRegistryPortType.class);
     }
 
+    @Bean
+    public BCeIDServiceSoap bCeIDServiceSoap() { return getPort(Clients.BCEID, BCeIDServiceSoap.class); }
```

Comment:
question again about consistent naming style
```
BCeIDServiceSoap()
```

Reply:
or is this because java methods should always start with lowercase?

Reply:
Yes to the lower case for java methods.